src/components/Label/Label.tsx

```diff
@@ -5,18 +5,18 @@ import { InfoBox } from '@/components/InfoBox/InfoBox';
 
 interface Props {
   name: string | JSX.Element;
-  description: string | JSX.Element;
+  info?: string | JSX.Element;
```

Comment:
Should this be React.ReactNode instead of just JSX.Element?

Reply:
Probably. IIRC, we should always use ReactNode rather than JSX.Element, right?  

We have been a bit sloppy in that case, and should fix this throughout the code base. Would prefer to leave this to after this PR is merged